src/remark-image-size.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path';
+
+import getImageSize from 'image-size';
+import {visit} from 'unist-util-visit';
+
+/**
+ * appends the image size to the image url as a hash e.g. /img.png -> /img.png#100x100
```

Comment:
@mjq-sentry What do you think about this trick?

Reply:
@mjq-sentry would appreciate you taking a look when you have a chance!

Reply:
Doing this as a `data-` attribute might be another option and a little less hacky, but I'm okay with trying this for now. The benefits outweigh the drawbacks in the short term.